<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 149, 148);
font: 75 18pt &quot;Agency FB&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="datos">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 18pt &quot;Agency FB&quot;;
border :5px solid black;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_com">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 18pt &quot;Agency FB&quot;;
border :5px solid black;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_accion">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>CONECTAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>COM:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_activarLuz">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>LUZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ActivarPuerta">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Puerta</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
